monitoring/ceph-mixin/README.md

```diff
@@ -0,0 +1,40 @@
+## Ceph mixin
+
+The ceph-mixin generates both grafana dashboards and alerts with Jsonnet.
+You can find all the generated files in the `output` directory and you can
+run `tox -egrafonnet-fix` to re-generate them.
+
+The ceph-mixin depends on some Jsonnet third party libraries located in the
+`vendor` directory. To update those libraries you will have to install
+[jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler) and run
+the command `jb update`.
+
+### Grafana dashboards for Ceph
+
+Here you can find a collection of [Grafana](https://grafana.com/grafana)
```

Comment:
What does "here" mean? Where can the user find the dashboards, exactly?

And which user is being addressed here? A reader of a web page, or someone reading RPM documentation?

How do the dashboards get installed on a system?

Reply:
This is basically a copy/paste of the old `monitoring/grafana/dashboards` README. It probably makes more sense in the old path because there was the JSON dashboard directly available in the directory. I will try to write something that makes more sense in the current context.

Dashboards are installed with the package `ceph-grafana-dashboards` AFAIU but they can also be generated from jsonnet by a user. I will try to reflect that in the README as well :+1:.

Reply:
I rephrased some sentence it should be a bit better. I think this would be more addressed to a dev/someone digging up in the source code.